.pre-commit-config.yaml

```diff
@@ -0,0 +1,79 @@
+#############################################################################################
+# pre-commit is a "framework for managing and maintaining multi-language pre-commit hooks"  #
+# see https://pre-commit.com/#intro for an introduction to pre-commit                       #
+#                                                                                            #
+# - pre-commit hooks are run on staged files just before a commit or a push                 #
+# - to run a given hook on any file at any time, run the following on the command line:     #
+#    pre-commit run <HOOK_ID> --files <FILE1> <FILE2> ..., e.g.,                            #
+#    pre-commit run autopep8 --files mal_formatted_file.py                                  #
+#############################################################################################
+exclude: '^$'
+# "exclude: '^$'" specifies the global file exclude pattern; '^$' excludes
+
+fail_fast: false
+# "fail_fast: false" means pre-commit continues to run hooks after the first failure
+
+repos:
+    -   repo: https://github.com/pre-commit/mirrors-mypy
+        rev: v0.971
+        hooks:
+            -   id: mypy  # Run mypy type checker; note: runs with flag --missing-imports
+                exclude: ondewo
+                args: [ --config-file=mypy.ini ]
+                # Use language: system to run Mypy git repo's existing virtual environment
+                # instead of mypy's own virtual environment. Necessary to find types-.* packages
+                # Replace language: python in your hook definition with language: system.
+                # Remove the additional_dependencies line and install Mypy into your
+                # environment directly. Now, Pre-commit will not create a separate virtualenv
+                # for the hook and will run it in whatever environment you happen to be in when
+                # you run git commit or pre-commit run => https://jaredkhan.com/blog/mypy-pre-commit
+                language: system
+                # use require_serial so that script
+                # is only called once per commit
+                require_serial: true
+                # Print the number of files as sanity-check
+                verbose: true
+                types: [ python ]
+
+    -   repo: https://github.com/pre-commit/mirrors-autopep8
+        rev: v1.6.0
+        hooks:
+            -   id: autopep8  # Perform pep8 auto-formatting
+                args: [ --max-line-length=120, --in-place ]
+
+    -   repo: https://github.com/pre-commit/pre-commit-hooks
+        rev: v4.3.0
+        hooks:
+            -   id: check-added-large-files  # Prevent giant files from being committed
```

Comment:
What happened to `    - id: no-commit-to-branch
      args: [--branch, master, --branch, develop]`?